### PR TITLE
Adding an alias(custom-ca-cert) for proxy-cert param for connectedK8s

### PR DESCRIPTION
**Pull request by pmpMsft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], arg_group='Proxy', help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], arg_group='Proxy', help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
-        c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well)')
+        c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well)')
```
Comment (bganapa) [inline src/connectedk8s/azext_connectedk8s/_params.py:55]: --custom-ca-cert ?

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], arg_group='Proxy', help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], arg_group='Proxy', help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
-        c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
+        c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-ca-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy or custom CA certificate')
```
Comment (bganapa) [inline src/connectedk8s/azext_connectedk8s/_params.py:36]: or custom certificate authority

File: src/connectedk8s/azext_connectedk8s/_params.py (hunk 1 above)
Comment (bganapa) [reply src/connectedk8s/azext_connectedk8s/_params.py:55]: 'Path to the any additional certificate file (for proxy as well)', the help text here could be same as the previous one -> 'Path to the certificate file for proxy or custom CA certificate'

**Comment by yonzhan** [comment]
connectedk8s

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], arg_group='Proxy', help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], arg_group='Proxy', help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
-        c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well)')
+        c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-ca-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well) or custom Certificate Authority certificate')
```
Comment (bganapa) [inline src/connectedk8s/azext_connectedk8s/_params.py:55]: lets change the whole help text from 'Path to the any additional certificate file (for proxy as well) or custom Certificate Authority certificate' to 'Path to the certificate file for proxy or custom Certificate Authority' . sorry for nitpicking :)

File: src/connectedk8s/azext_connectedk8s/_params.py [hunk 4]
```diff
@@ -33,7 +33,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], arg_group='Proxy', help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], arg_group='Proxy', help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
-        c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
+        c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-ca-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy or custom Certificate Authority')
```
Comment (sarathys) [inline src/connectedk8s/azext_connectedk8s/_params.py:36]: do we need a new parameter in case user wants to pass in proxy cert and custom ca cert?

Comment (bganapa) [reply src/connectedk8s/azext_connectedk8s/_params.py:36]: there can be multiple certs combined in one .crt file. Agents code base just accepts one file now. If there is a need, the user will be combining multiple certs in to one file

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**